sega_genesis_checksum_utility_for_python_3.py

```diff
@@ -25,46 +25,46 @@ def main():
                                             'Genesis ROM binary.')
   args = parser.parse_args()
 
-  print 'Reading checksum from file...'
-  with open(args.path[0], 'r+b') as genesis_file:
+  print('Reading checksum from file...')
+  with open(args.path[0], 'rb') as genesis_file:
```

Comment:
Doesn't this need to be `r+b` for the `write_checksum_to_header`?

Otherwise it's only open for reading.

Reply:
Yes, it caught it. It should be r+b to run on any os. I am changing it.

Reply:
Sorry for not being more clear.

This is not a problem with the code itself, but just how it's formatted.

Can we remove the spaces on the wrapped line so that the quotes of the text are aligned?

e.g. https://i.imgur.com/xjqEBTn.jpg